src/k2/dto/Persistence.h

```diff
@@ -77,5 +77,55 @@ struct PlogSealResponse {
     K2_DEF_FMT(PlogSealResponse, sealedOffset);
 };
 
+struct PlogStatusRequest {

```

Comment:
should be PlogGetStatusRequest and PlogGetStatusResponse (note the action/verb in the name)

Reply:
changed